modules/core/package.json

```diff
@@ -39,5 +39,8 @@
     "math.gl": "^3.4.2",
     "mjolnir.js": "^2.5.0",
     "probe.gl": "^3.3.1"
+  },
+  "devDependencies": {
```

Comment:
What does this do for the users? `devDependencies` are not installed when you install this package from npm.